File: 프로그래머스/1/12954. x만큼 간격이 있는 n개의 숫자/x만큼 간격이 있는 n개의 숫자.java
class Solution {
    public long[] solution(int x, int n) {
        long[] answer = new long[n];
        for (int i=0; i<n; i++){
            long point= (long) x*((long) i+1);
            answer[i]= point;
        }
        return answer;
    }
}

File: 프로그래머스/1/12954. x만큼 간격이 있는 n개의 숫자/x만큼 간격이 있는 n개의 숫자.kt
class Solution {
    fun solution(x: Int, n: Int): LongArray {
        var answer = longArrayOf()
        var list= ArrayList<Long>()
        if( x in -10000000 .. 10000000 && n in 1 .. 1000){
            for (i in 1 .. n){
                list.add(i.toLong()*x)
            }
        }
        answer=list.toLongArray()
        return answer
    }
}